fieldservice_person_salary/views/fsm_order_views.xml

```diff
@@ -16,5 +16,5 @@
             </xpath>
         </field>
     </record>
-  
```

Comment:
Please remove this change (that blank should be removed, but better not in this commit, since this file is not linked with any required change)